libmscore/style.cpp

```diff
@@ -593,7 +593,7 @@ static const StyleType styleTypes[] {
       { Sid::barGraceDistance,        "barGraceDistance",        Spatium(1.0) },
       { Sid::minVerticalDistance,     "minVerticalDistance",     Spatium(0.5) },
       { Sid::ornamentStyle,           "ornamentStyle",           int(MScore::OrnamentStyle::DEFAULT) },
-      { Sid::spatium,                 "Spatium",                 SPATIUM20 },
+      { Sid::spatium,                 "Spatium",                 24.8 },
```

Comment:
At several places <code>SPATIUM20</code> is used. With this change <code>SPATIUM</code> and <code>Sid::spatium</code> are not the same anymore.
Are we sure this doesn't cause any problems because both <code>SPATIUM</code> and <code>Sid::spatium</code> are expected to be same or one is used where the other should be?

Reply:
I limited myself to the changes here at Simon's request, let's see how this affects those elements that do not use it through the style, for some reason, and if something goes wrong, we will replace the value of spatium20 in the future

Reply:
If so, we should fix those places.  But while Sid::spatium might have *defaulted* to SPATIUM20, certainly we always support changing that value, and a great many scores do, and overall, things work.  We do find and fix bugs from time to time where things don't scale properly, but I don't remember any cases where it was on account of confusing these two values - normally just because we were forgetting to account for this at all, like in some particular layout(), draw(), or mag() function.  Or we were missing it in spatiumChanged().

Reply:
Yeah, perhaps, Probably we'll need to replace direct usage of SPATIUM20 with styleD(Sid::Spatium) in the next PRs